app/assets/javascripts/components/bulkUploadImport.jsx

```diff
@@ -424,7 +463,8 @@ class BulkUploadImport extends React.Component {
   render() {
     return (
       <div>
-        { this.state.imported ? this.renderBulkUploadSubmitForm(): this.renderBulkUploadImportForm() }
+        { this.state.imported ? this.renderBulkUploadSubmitForm() : null }
```

Comment:
what's the reason of separating them into different lines?

Reply:
having them on both sides of the ternary operator was throwing a DOM error poiting to hiding and not completely removing the component(after rendering), so I decided to either show or return null for both components as opposed to hiding. @yunfangjuan